src/modules/previewpane/powerpreview/powerpreview.cpp

```diff
@@ -79,20 +135,24 @@ void PowerPreviewModule::set_config(const wchar_t* config)
 // Enable preview handlers.
 void PowerPreviewModule::enable()
 {
-    registry_and_elevation_check_wrapper([this]() {
-        for (auto fileExplorerModule : this->m_fileExplorerModules)
-        {
-            if (fileExplorerModule->GetToggleSettingState())
-            {
-                // Enable all the modules with initial state set as true.
-                fileExplorerModule->Enable();
-            }
-            else
+    // Should only be done for old settings as it is already done for new settings in the constructor.
+    if (!UseNewSettings())
+    {
+        registry_and_elevation_check_wrapper([this]() {
+            for (auto fileExplorerModule : this->m_fileExplorerModules)
             {
-                fileExplorerModule->Disable();
+                if (fileExplorerModule->GetToggleSettingState())
+                {
+                    // Enable all the modules with initial state set as true.
+                    fileExplorerModule->Enable();
+                }
+                else
+                {
+                    fileExplorerModule->Disable();
+                }
```

Comment:
nit:  It might be a good idea to put this code in a method as it's duplicated by the constructor